src/rantanplan/core.py

```diff
@@ -17,17 +22,28 @@
 paroxytone_re = re.compile("([aeiou]|n|[aeiou]s)$", re.I | re.U)  # checks if a str ends in unaccented vowel/N/S
 
 letter_clusters_re = re.compile(r"""
-    ([iuü]h[iuü])|                                            # 1: vowels
-    ([aáeéíoóú]h[iuü])|                                       # 2: open vowels
-    ([iuü]h[aáeéíoóú])|                                       # 3: closed vowels
-    ([a-záéíóúñ][bcdfghjklmnñpqrstvxyz][hlr][aáeéiíoóuúü])|   # 4: liquid and mute consonants (adds hyphen)
-    ([bcdfghjklmnñpqrstvxyz][hlr][aáeéiíoóuúü])|              # 5: any char followed by liquid and mute consonant
-    ([a-záéíóúñ][bcdfghjklmnñpqrstvxyz][aáeéiíoóuúüï])|        # 6: non-liquid consonant (adds hyphen)
-    ([aáeéíoóú][aáeéíoóú])|                                   # 7: vowel group (adds hyphen)
-    (ü[eií])|                                                 # 8: umlaut 'u' diphthongs
-    ([aeiou][äëïöü])|                                         # 9: Explicit hyatus with umlaut vowels, first part
-    ([äëïöü][a-z])|                                           #10: Explicit hyatus with umlaut vowels, second part
-    ([a-záéíóúñ])                                             #11: any char
+    # 1: vowels
+    ([iuü]h[iuü])|
```

Comment:
Would this be something like h diphthong with weak vowels?

Reply:
Indeed